optuna/samplers/_tpe/sampler.py

```diff
@@ -233,6 +313,40 @@ def _split_observation_pairs(
         above = np.asarray([v for v in above if v is not None], dtype=float)
         return below, above
 
+    def _split_multivariate_observation_pairs(
+        self,
+        config_vals: Dict[str, List[Optional[float]]],
+        loss_vals: List[Tuple[float, float]],
+    ) -> Tuple[Dict[str, np.ndarray], Dict[str, np.ndarray]]:
+
+        config_vals = {k: np.asarray(v, dtype=float) for k, v in config_vals.items()}
+        loss_vals = np.asarray(loss_vals, dtype=[("step", float), ("score", float)])
+
+        # TODO(kstoneriv3): Change of the order of 1. exclusion of `None` and 2. splitting might
+        # be needed. Independent sampler in `TPESampler` first splits the observations and
+        # then exclude `None`.
```

Comment:
I think we can remove this TODO comment because the current implementation is reasonable. As another option, how about removing `TODO` and rewriting the comment as follows? "We firstly exclude `None` and then split observations. This order is the opposite of the `independent _sample`."

Reply:
I agree. I will change the comment as you suggested.